internal/pkg/composable/providers/kubernetessecrets/kubernetes_secrets.go

```diff
@@ -136,7 +136,7 @@ func (p *contextProviderK8SSecrets) Fetch(key string) (string, bool) {
 		return "", false
 	}
 	if len(tokens) != 4 {
-		p.logger.Warn("Invalid secret key format: ", key, ". Secrets should be of the format kubernetes_secrets.namespace.secret_name.value")
+		p.logger.Warnf(`Invalid secret key format: "%s". Secrets should be of the format kubernetes_secrets.namespace.secret_name.value`, key)
```

Comment:
fyi `%q` is a formatting directive to quote a string

Reply:
TIL thanks @michel-laterman , I think I am gonna push this change

Reply:
just pushed a commit about it, please feel free to unresolve if you spot anything else 🙂